packages/coordinator/src/screens/manageAppointmentsScreen/ManageAppointmentsScreen.tsx

```diff
@@ -156,6 +171,8 @@ export default function ManageAppointmentsScreen({
         onApproved={popupData.onApproved}
         onClose={() => setPopupData(emptyPopupData)}
       />
+      <PopupFlashMessage message={flashMessage} showFlash={showFlash} />
```

Comment:
I think the message can be hard coded here, as  there's only one option for it - "הפרטים הועתקו בהצלחה"